src/main/java/frc/robot/Constants.java

```diff
@@ -62,4 +57,32 @@
     public static final int kAButton = 1;           // Button for hopper
     public static final int kRightBumperButton = 5; // Button for intake 
     public static final int kLeftBumperButton = 6;  // Button to reverse intake
+    public static final int kXbutton = 3; // Button for Shooter
+    public static final int kDownbutton = 4; // Button for shooter mode
+    public static final int kUpbutton = 5; // Button for shooter mode
+
+    // Preset aim constants
+    public static final int kLOWRPM = 0; // RPM that the LOW aimMode winds to
+    public static final double kLOWAngle = 0.0; // Angle that the LOW aimMode adjusts to
+    public static final int kLAUNCHRPM = 0; // RPM that the LAUNCH aimMode winds to
+    public static final double kLAUNCHAngle = 0.0; // Angle that the LAUNCH aimMode adjusts to
+    public static final int kTARMACRPM = 0; // RPM that the TARMAC aimMode winds to
+    public static final double kTARMACAngle = 0.0; // Angle that the TARMAC aimMode adjusts to
+
+    // Shooter Constants
+    public static final int kShooterID = 1; // ID of the shooter
+    public static final int kHoodID = 2; // ID of the hood;
+    public static final double kShooterHeight = 3;
+    public static final double kHighHeight = 8.0 + 8.0/12.0; // Height of the high goal in ft from the carpet
+    public static final double kLowHeight = 5.0 + 7.75/12.0; // Height of the low goal in ft from the carpet
+    public static final double kAirboneTime = 1.2;
+    public static final double kGravity = 32;
+    public static final double kLLHeight = 1.5; // Height of the limelight in ft from the carpet
+    public static final double kLLAngle = 30.0; // Angle that the limelight is mounted at
+    public static final double kGearRatioIn = 34;
+    public static final double kGearRatioOut = 14;
+    public static final double kGearDiameter = 4; // Gear diametter in inches
+    public static final double kBallFlywheelratio = 2;
+    public static final int kShooterCargoID = 6;
```

Comment:
Conflict with motor ID in drive base (kDriveRightFront)

Reply:
Ignore this, drive base team is removing ID 6 now that we only have 4 drive motors